Src/FluentAssertions/Equivalency/Field.cs

```diff
@@ -26,6 +27,12 @@ public Field(Type reflectedType, FieldInfo fieldInfo, INode parent)
             Name = fieldInfo.Name;
             Type = fieldInfo.FieldType;
             RootIsCollection = parent.RootIsCollection;
+
+            bool isNonBrowsable =
```

Comment:
🔧 I'm not 100% sure it is really needed, but I would use a lazy-evaluated solution to not add more reflection if it isn't really needed. Same for the `Property` member.

Reply:
Easy enough to do. :-)